flipperdevices/flipperzero-firmware
# [FL-3783] Asynchronous Infrared remote manipulation

# What's new

- Load, delete, rename, move whole remotes and individual buttons in a separate thread in order to not stall the UI.

# Verification 

1. Install both the firmware and the Infrared application.
2. Go to `Infrared -> Universal Remotes -> TVs`
3. Mash the buttons while the busy animation is playing. The app should ignore the input and successfully load the file.
4. Perform all other potentially lengthy operations: Rename the signals, move the signals. Mash the buttons during the busy animation as well (Works best on large remotes with hundreds of signals). The result should be the same as in step 3.
5. Test all other features to make sure that no regressions are present.

# Checklist (For Reviewer)
